test/tstDistributedTreeNearest.cpp

```diff
@@ -274,6 +274,8 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(unique_leaf_on_rank_0_nearest, DeviceType,
                               ARBORX_DEVICE_TYPES)
 {
   using ExecutionSpace = typename DeviceType::execution_space;
+  using Point = ArborX::Point<3>;
+  using Box = ArborX::Box<3>;
```

Comment:
Do we want to be explicit here and say float?

Reply:
Not right now. I'll do that once we switch distributed test to run both.